development/lint/electron.js

```
@@ -19,3 +20,25 @@ try {
   );
   exit(1);
 }
+
+// Check if APP_NAME is correctly set to "OneKey Wallet"
+  const distAppPath = path.join(desktopPath, 'app', 'dist', 'app.js');
+  console.log(distAppPath);
+  if (!fs.existsSync(distAppPath)) {
+    throw new Error(`Build output file not found: ${distAppPath}`);
+  }
+  const expectedAppName = 'APP_NAME = "OneKey Walletaaaa"';
```

ESLint

[error] Delete `··`

(prettier/prettier)